src/main/java/com/cs544/project/controller/sysadmin/StudentController.java

```diff
@@ -0,0 +1,47 @@
+package com.cs544.project.controller.sysadmin;
+
+import com.cs544.project.domain.Student;
+import com.cs544.project.service.StudentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/sys-admin/students")
+public class StudentController {
```

Comment:
Based on our previous pattern and convention, this should be a `SystemAdminViewController` with @RequestMapping("/sys-admin").

Then we will have a method for each CRUD operation on students e.g. @GetMapping("/students"), @GetMapping("/students/{studentId}"), @PostMapping("/students") etc

Reply:
@bibangamba It's better to separate individual resource controllers into their own classes. I disagree with adding all resources crud operation into one single controller.